src/components/layertree/LayerTree.test.js

```diff
@@ -124,51 +133,77 @@ describe('LayerTree', () => {
   });
 
   describe('when a radio item is checked', () => {
-    test('expands the item, uncheck and collapse all the siblings, and apply defaults value to the children.', () => {
-      const data2 = {
-        rootId: 'root',
-        items: {
-          root: {
-            children: ['1', '2', '3'],
-          },
-          '1': {
-            type: 'radio',
-            children: ['1-1', '1-2'],
-          },
-          '2': {},
-          '3': {
-            isChecked: true,
-            isExpanded: true,
-            type: 'radio',
-          },
-          '1-1': {
-            defaults: {
+    test(
+      'expands the item, uncheck and collapse all the siblings and their ' +
+        'radio siblings children, and apply defaults value to the children.',
```

Comment:
expands the item, uncheck and collapse all the radio siblings and their children, then apply default to the item's children ...